cvpy/tests/test_cvat_project.py

```diff
@@ -288,30 +288,45 @@ def test_cvat_project_save_no_caslib_no_relative_path(self):
 
     def test_cvat_project_resume(self):
 
-        # Resume a test project
-        cvat_project = CVATProject.resume(project_name='MyDemoProject', cas_connection=self.cas_connection,
+        # Inner-class used for testing CVAT scenerios
+        def test_cvat_project_resume_assertions(cvat_project):
+            # Verify all project attributes are set correctly
+            assert cvat_project.project_version == 1
+            assert cvat_project.project_name == 'MyDemoProject'
+            assert cvat_project.project_id == 285
+            assert cvat_project.annotation_type == AnnotationType.OBJECT_DETECTION
+            assert cvat_project.credentials.username is None
+            assert cvat_project.credentials.password is None
+            assert len(cvat_project.labels) == 2
+
+            for label in cvat_project.labels:
+                if label.name == 'Mountain':
+                    assert label.color == 'orange'
+                elif label.name == 'Person':
+                    assert label.color == 'green'
+                else:
+                    assert False
+
+            for task in cvat_project.tasks:
+                assert task.image_table_name == 'cas_table_encoded'
+                assert task.image_table.table.tableinfo().TableInfo.Rows.values[0] == 5
+
+        # Create variables for all cvat resume scenerios
+        cvat_project_relativepath_caslib = CVATProject.resume(project_name='MyDemoProject', cas_connection=self.cas_connection,
```

Comment:
Break each of these scenarios out into a separate test case. So there should be four test cases. 

def test_cvat_project_resume(self) == caslib and path are both specified
def test_cvat_project_resume_default_caslib(self) == caslib is default, path is specified
def test_cvat_project_resume_default_path(self) == caslib is specified, path is default
def test_cvat_project_resume_default_caslib_default_path(self) == both caslib and path are default

Each of the above functions will call the _verify_cvat_project_attributes(cvat_project) function.